msteams-platform/concepts/deploy-and-publish/appsource-publishing-guidance/prepare-app/overview.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Submission checklist 
+description: The checklist to use before publishing your Microsoft Teams app to AppSource
+keywords: teams publish store office publishing checklist
+---
+
+Teams app approval is a free service provided by the Microsoft Teams group that verifies that your app works as described, contains all appropriate metadata, and provides content that would be valuable to an end user.
+
+For your Teams app to be approved, your app must meet the following requirements and guidelines:
+
+* **Distribution Method**: Make sure your app is meant for a store. There are [other options](msteams-platform\concepts\deploy-and-publish\overview.md)
+* **App Functionality**: Your app meets [App Functionality guildeline](msteams-platform\concepts\deploy-and-publish\appsource-publishing-guidance\prepare-app\app-functionality-requirement.md) to deploy your app without submitting to AppSource.
+* **App Detail Page**: Your app meets [App Detail Page Checklist](~/concepts/deploy-and-publish/appsource-publishing-guidance/prepare-app/app-detail-page-checklist.md)
+* **Frequently Failed Cases**: Pay extra attention to these [Frequently Failed Cases]((~/concepts/deploy-and-publish/appsource-publishing-guidance/prepare-app/frequently-failed-cases.md)) to improve your app submission time.
+* **Manifest**: Your app manifest pacakge pass the [Manifest Package Checklist](msteams-platform\concepts\deploy-and-publish\appsource-publishing-guidance\prepare-app\office-store-checklist.md) and [Manifest Checker in App Studio]()
+* **Testing**: You have fully [tested and debugged your app](https://docs.microsoft.com/en-us/microsoftteams/platform/concepts/build-and-test/debug).
+* It must pass all current [AppSource validation policies](https://dev.office.com/officestore/docs/validation-policies) for Teams tabs and bots. Please note that these policies are subject to change.
+* Testing Notes: Include proper [Test Notes for validation](#Test-Notes-for-validation/approval)
```

Comment:
"Testing Notes" is another confusing name. Probably should be something like "Test plan and credentials" or "Test cases and credentials", or maybe "Test scripts.."